src/Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -134,7 +142,7 @@ private function doExecute(InputInterface $input, OutputInterface $output): int
             $input->getOption('purge-with-truncate'),
         );
         $executor = new ORMExecutor($em, $purger);
-        $executor->setLogger(static function ($message) use ($ui): void {
+        $executor->setLogger($this->logger ?? static function ($message) use ($ui): void {
             $ui->text(sprintf('  <comment>></comment> <info>%s</info>', $message));
         });
```

Comment:
Passing a closure as logger is deprecated. We could use an anonymous class instead that extends PSR-3's `AbstractLogger`.

Reply:
I don't understand your comment, but I see I made a big mistake here: I didn't check for support of the PSR-3 logger.

Reply:
I don't see a straightforward way to detect it. I think I'll just bump the dependency.

Reply:
Ah I understood, Copilot gave me an example implementation. So… I guess I should remove the new `$logger` property entirely then.